smoke-testing/src/test/java/games/strategy/engine/data/GameTestUtils.java

```diff
@@ -46,6 +47,7 @@ public static void setUp() throws IOException {
       System.setProperty("user.home", tempHome.toString());
     }
     ClientSetting.initialize();
```

Comment:
I think the initialize is redundant if setPreferences is called afterwards, right?

Reply:
It has some locale logic:
```
    final Locale defaultLocale = Locale.getDefault();
    if (!I18nResourceBundle.getMapSupportedLocales().contains(defaultLocale)) {
      Locale.setDefault(Locale.US);
    }
```

Reply:
Ah I see. Didn't realize this was initialized there (imo it should be initialized independent of this class, does not have much to do with the client settings).
The error message is still treating them as equivalent though:
`ClientSetting framework has not been initialized. Did you forget to call ClientSetting#initialize() in production code or ClientSetting#setPreferences() in test code?`
So either the message should be updated, or the code moved somewhere else to be called explicitly.

Reply:
Fair enough. Since there were only two other call sites of `initialize()`, I just inlined that locale logic into them.

The test worked fine without it, so I just left it out.